src/Net.jl

```diff
@@ -261,32 +301,50 @@ function read_col(sock::ClickHouseSock, num_rows::VarUInt)::Column
     name = chread(sock, String)
     type_name = chread(sock, String)
 
-    decode_type_name, enum_def = if startswith(type_name, "Enum")
-        parse_enum_def(type_name)
+    trim_type_name = if startswith(type_name, "Nullable")
+        type_name[10:end-1]
     else
-        type_name, nothing
+        type_name
     end
-
-    decode_type = try
-        COL_TYPE_MAP[decode_type_name]
-    catch exc
-        if exc isa KeyError
-            error("Unsupported data type: $(decode_type_name)")
+        
+    decode_type_name, enum_def = if startswith(trim_type_name, "Enum")
+        parse_enum_def(trim_type_name)
+    else
+        trim_type_name, nothing
+    end
+        
+    if startswith(decode_type_name, "FixedString(")
+        decode_type = decode_type_name |> Meta.parse |> eval
```

Comment:
This introduces an [RCE](https://en.wikipedia.org/wiki/Arbitrary_code_execution) vulnerablity where a malicious server could execute arbitrary code on the client. Please parse this manually, e.g. with regular expressions.